meta/applications/yc_s15.md

```diff
@@ -0,0 +1,402 @@
+out anyway in one and a half instead.
+
+**Please tell us in one or two sentences about the most impressive thing other
+than this startup that you have built or achieved.**
+
+In my freshman year of high school, I joined Run Games as the first employee
+and brought our game from an early prototype to launching in the App Store and
+getting over one million users. We reached #5 in the top free charts, #1 in
+games, #1 in sports, #1 in arcade, and were featured by Apple on the front page
+of the App Store.
+
+Other potentials:
+
+* PennApps, youngest winner
+* CTO of Yo
+
+**Will you live in the Bay Area June through August 2015, if we fund you?**
+
+Yes.
+
+**Will you commit to working exclusively on this project for the next year?**
+
+Yes.
+
+**Do you have any other commitments between June through August 2015
+inclusive?**
+
+No.
+
+**Do you have any commitments in the future and if so what?**
+
+No.
+
+#### Jonathan Leung
+
+**Name**
+
+Jonathan Leung
+
+**Email Address**
+
+jonathan@hackedu.us
+
+**Age**
+
+24
+
+**Gender**
+
+Male
+
+**Schools, Degrees and Years of Graduation**
+
+University of Pennsylvania, BSE in Computer Engineering, 2013
+The Haverford School, GED, 2009
+
+**Employers, Position / Titles and Dates**
+
+```
+All Star Code
+Curriculum Developer And Facilitator
+Mar - Aug 2014
+
+General Assembly
+Instructor for the 12-Week Ruby on Rails Course
+Sep - Dec 2013
+
+Learners' House (bit.ly/learnershouse)
+Creator
+Dec 28 - Jan 7, 2014
+
+PennApps
+Lead Organizer
+2010 - 2012
+
+Penn Rent-A-CTO
+Co-Founder and Consultant
+2011 - 2013
+
+Perceptual Networks
+Full Stack Intern
+Jun - Aug 2012
+
+Gobbler
+Full Stack Intern
+Jun - Aug 2011
+```
+
+**Personal website**
+
+http://jonl.org/
+
+**Github URL**
+
+https://github.com/jonleung
+
+**Linkedin URL**
+
+www.linkedin.com/in/jonathanjleung
+
+**Facebook URL**
+
+http://fb.jonl.org
+
+**Twitter URL**
+
+https://twitter.com/jonathanjleung
+
+**Please tell us about the time you most successfully hacked some
+(non-computer) system to your advantage.**
+
+While organizing PennApps in 2011, I thought it would be compelling for
+participants for participants if we could pay for their transportation to our
+hackathon. I thought that we would be able to convince MegaBus to be our
+"official transportation sponsor" because hackathon goers fit their demographic
+of "young persons likely to want travel between large cities cheaply while
+using wi-fi". They were interested but hesitant because they didn't want to
+associate themselves with all the "hackers" and all the "hacking" that was
+happening in the news.
+
+To alleviate their concerns, I innocently sent them the link to the Wikipedia
+entry for Hackathon--after I added this: 'Although "hack" is in the root of
+hackathon, hackathons have nothing to do with malicious "hacking" done by
+hackers, rather the spirit of a hackathon is to collaboratively build programs
+and applications.'
```

Comment:
Could be clearer. The previous one was good because the first sentence basically tells the reader everything they need to know.

Keep in mind that people will be skimming this.